### DISCUSSION TITLE
In the latest version Prometheus Exporter `func (*Exporter) ServeHTTP`  was removed  , how can i to expoter my metric use http server ?

### DISCUSSION THREAD

**Discussion by Lucareful** [open-telemetry/opentelemetry-go#5974] [Q&A] [answered]
`go.opentelemetry.io/otel/exporters/metric/prometheus`  v0.21.0 
![image](https://github.com/user-attachments/assets/7cbf20ca-217b-4c9a-9402-8999056fa0d4)

why remove this func in the latest version? 😳

**Comment by dmathieu** [comment, marked as answer]
See the prometheus example:
https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/examples/prometheus/main.go#L88

You should be using `promhttp` as the handler to make metrics available. The prometheus exporter will pass data to it.

**Reply by pellared** [reply]
I think it also worth to notice that it was also not removed in the latest version, but more than 2 years ago in https://pkg.go.dev/go.opentelemetry.io/otel/exporters/prometheus@v0.32.0 (times run quickly 😭 )

**Reply by Lucareful** [reply]
Haha, it's the version I'm using that is lagging behind, thank you guys for so long maintenance and dedication, it's a very meaningful thing!!! The time spent is worth it 😎
